CHANGELOG.md

```diff
@@ -88,7 +88,9 @@
   https://github.com/googleapis/google-cloud-cpp/issues/5976.
 </details>
 
-## v1.36.0 - TBD
+## v1.37.0 - TBD
+
+## v1.36.0 - 2022-02
```

Comment:
I did not think of these at the time, but please change:

`https://github.com/googleapis/google-cloud-cpp/pull/8095` -> `[#8095](https://github.com/googleapis/google-cloud-cpp/pull/8095)`
`https://github.com/googleapis/google-cloud-cpp/pull/8099` -> `[#8099](https://github.com/googleapis/google-cloud-cpp/pull/8099)`

Reply:
Or we could just say "as part of this release" instead of quoting specific PRs.